docs/modules/ROOT/pages/native-memory.adoc

```diff
@@ -0,0 +1,51 @@
+= Native Memory
+:description: Native Memory Store is designed as a pluggable memory manager which enables multiple memory stores for different data structures. These memory stores are all accessible by a common access layer that scales up to massive amounts of the main memory on a single JVM by minimizing the GC pressure. Native Memory Store enables predictable application scaling and boosts performance and latency while minimizing garbage collection pauses.
+
+{description}
+
+This topic explains how to configure native memory. It also brings together example of how you can configure your data structure with Native Memory.
+
+== Configuring Native Memory
+
+Below are the configuration options for the Native Memory feature.
+
+[cols="25%m,75%a"]
+|===
+|Field|Description
+
+|`allocatorType`
+| AllocatorType specifies one of 2 types of mechanism for allocating memory.
```

Comment:
```suggestion
| Specifies the mechanism for allocating memory; either XXXX or XXXX.
```
It makes sense to give both options if they are available, and to explain what they are used for.